timelock-server/build.gradle

```diff
@@ -54,5 +54,6 @@ dependencies {
     testCompile group: 'com.squareup.okhttp3', name: 'okhttp'
 
     integTestCompile group: 'io.dropwizard', name: 'dropwizard-testing'
-    integTestCompile group: 'com.palantir.remoting3', name: 'http2-agent'
+    integTestCompile group: 'com.palantir.conjure.java.runtime', name: 'jetty-http2-agent'
+    integTestCompile group: 'org.mortbay.jetty.alpn', name: 'jetty-alpn-agent'
```

Comment:
this as an artifact of my debugging and I think we should be able to get rid of it